jsapi/externs/luxx.js

```diff
@@ -150,3 +157,16 @@ luxx.MousePositionOptions.prototype.srs;
  * @type {Element|string}
  */
 luxx.MousePositionOptions.prototype.target;
+
+/**
+ * Configuration for the controls.
+ * @typedef {{ target: (Element|string) }}
+ */
+luxx.ControlOptions;
+
+/**
+ * The container for the control UI component, either the element itself or
+ * the `id` of the element.
+ * @type {Element|string}
+ */
+luxx.ControlOptions.prototype.target;
```

Comment:
There are others control with `target` as single property for the moment. I wonder what’s better between a generic one (I don’t think you can extend typedef) or, specific one to be easily extended.

I generic one is chosen (`ControlOptions`), at least `BgSelectorOptions` should be updated to use it.